st2common/bin/st2ctl

```diff
@@ -43,19 +43,20 @@ function print_usage() {
     echo
     echo "Usage: st2ctl {reload, clean}"
     echo "optional arguments:"
-    echo "  --register-all                Register all."
-    echo "  --register-triggers           Register all triggers."
-    echo "  --register-sensors            Register all sensors."
-    echo "  --register-rules              Register all rules."
-    echo "  --register-runners            Register all runners."
-    echo "  --register-actions            Register all actions."
-    echo "  --register-aliases            Register all aliases."
-    echo "  --register-policies           Register all policies."
-    echo "  --register-configs            Register all configuration files."
-    echo "  --register-setup-virtualenvs  Create Python virtual environments for all the registered packs."
-    echo "  --register-fail-on-failure    Exit with non-zero if some resource registration fails. Deprecated. This is now a default behavior."
-    echo "  --register-no-fail-on-failure Don't exit with non-zero if some resource registration fails."
-    echo "  --verbose                     Output additional debug and informational messages."
+    echo "  --register-all                                Register all."
+    echo "  --register-triggers                           Register all triggers."
+    echo "  --register-sensors                            Register all sensors."
+    echo "  --register-rules                              Register all rules."
+    echo "  --register-runners                            Register all runners."
+    echo "  --register-actions                            Register all actions."
+    echo "  --register-aliases                            Register all aliases."
+    echo "  --register-policies                           Register all policies."
+    echo "  --register-configs                            Register all configuration files."
+    echo "  --register-setup-virtualenvs                  Create Python virtual environments for all the registered packs."
+    echo "  --register-setup-virtualenvs-force-recreation (Delete and re-)create Python virtual environments for all the registered packs."
```

Comment:
Thinking about the shorter version, how this would sound instead? :thinking: 

```diff
---register-setup-virtualenvs-force-recreation
+--register-setup-recreate-virtualenvs
```

Reply:
I like it especially because it's more readable. Done & pushed. 

I'll have to go to sleep now and will proceed with the testing tomorrow as well as looking into failing tests (if any).

Reply:
OOH! The bike shed needs another coat of paint :trollface: 

Let's shorten that even farther:
```diff
-  --register-setup-virtualenvs-force-recreation
+  --register-recreate-virtualenvs
```

Reply:
I'm resisting the recommendation to drop "register", even though re/creating virtualenvs doesn't actually "register anything. Alas, keeping "register" keeps the args consistent.

Reply:
Yeah, I like ``--register-recreate-virtualenvs`` since it's the shortest :D

IIRC, register prefix was added at some point in the past due to some oslo config limitations or similar, but I forget the details.

Reply:
Yes, there seems to be a limitation that requires the --register-xxx prefix. I can dig deeper into if needed but will focus on the failing tests and actual issues for now.